src/ft/mod.rs

```diff
@@ -409,9 +409,9 @@ impl FreeTypeRasterizer {
                     }
 
                     let pattern = font_pattern.clone();
-                    let key = self.face_from_pattern(&pattern, font_key)?.unwrap();
-
-                    return Ok(key);
+                    return self
+                        .face_from_pattern(&pattern, font_key)?
+                        .ok_or(Error::FontFaceLocationNotFound);
```

Comment:
I don't think we should return an error here. Doing so basically means that as soon as this font is encountered, no more fonts will be checked. We should just pretend that no glyph is inside this font by never returning its key. So just skipping to the next loop step should do the job.

Reply:
You are right. When returning the error, alacritty showed a '▯'. I changed it and now alacritty simply uses another font, which is way better.

I did run into some trouble with the borrow checker, so I had to clone the list of fallback fonts.